_data/matrix/Phoenix.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Phoenix"
+category: "Software Wallet (L2)"
+keyfeatures: "Phoenix is an easy-to-use non-custodial lightning wallet"
+homepage: "https://phoenix.acinq.co/"
+
+feature:
+  device: "Mobile"
+  os: "Android, iOS"
```

Comment:
In the past we had the issue that e.g. Phoenix had some features that were only live on Android but not on iOS or vice versa. How could we handle that situation in the future?

Reply:
This is indeed a bit of a curveball, I will need to mull on this.